src/permission-set.js

```diff
@@ -655,6 +759,37 @@ class PermissionSet {
   }
 
   /**
+   * @method loadGroups
+   * @param [options={}]
+   * @param [options.fetchGraph] {Function} Injected, returns a parsed graph of
+   *   a remote document (group listing). Required.
+   * @param [options.rdf] {RDF} RDF library
+   * @throws {Error}
+   * @return {Promise<PermissionSet>} Resolves to self, chainable
+   */
+  loadGroups (options = {}) {
+    let fetchGraph = options.fetchGraph
+    let rdf = options.rdf || this.rdf
+    if (!fetchGraph) {
+      return Promise.reject(new Error('Cannot load groups, fetchGraph() not supplied'))
```

Comment:
I'm not a fan of this practice of passing `fetchGraph` / `rdf` / `webclient` objects around to methods which need them. At the very, very least there should be a default implementation that's provided by the library.